.github/workflows/publish_ci.yml

```diff
@@ -8,27 +8,43 @@ on:
 jobs:
   build:
 
+    env:
+      GITHUB_TOKEN: '${{secrets.GITHUB_TOKEN}}'
```

Comment:
Putting secrets into globally available env vars is not the best idea ever.

Reply:
What is a better idea? Just access like this `${{secrets.GITHUB_TOKEN}}`?

Reply:
Yes, or only set the env var for specific steps if really necessary.

Reply:
Could you show how to do that?

Reply:
Setting an env var? Same as here but in the step definitions needing an env var. Although, I saw they already access the token through the secrets context so I don't know why you additionally put it into an env var for everyone to read.

Reply:
Looking at this more, I realize that the workflow-global env may not even have access to the `secrets` context so it's possible that you're setting the var to an empty string already.